src/SamplesApp/SamplesApp.Shared/MainPage.xaml

```diff
@@ -6,6 +6,15 @@
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
 	mc:Ignorable="d">
-
-	<controls:SampleChooserControl x:Name="sampleControl" />
+	
+	<Grid Padding="12">
+	 	<controls:SampleChooserControl x:Name="sampleControl" Visibility="Collapsed" />
+		<StackPanel>
```

Comment:
still in draft but commenting just in case :) Those seem like quick tests ^^

Reply:
Yep, to be removed :)